.gitignore

```diff
@@ -6,6 +6,7 @@
 *.dylib
 .DS_Store
 .idea
+examples/http-server/http-server
```

Comment:
I think I would add this gitignore inside the example itself. Here we only include very general ignores like the ones created by the IDE.